app/gui2/src/util/vec2.ts

```diff
@@ -25,7 +25,7 @@ export class Vec2 {
   scale(scalar: number): Vec2 {
     return new Vec2(this.x * scalar, this.y * scalar)
   }
-  distanceSquare(a: Vec2, b: Vec2): number {
+  static distanceSquare(a: Vec2, b: Vec2): number {
```

Comment:
`distanceSquared` to be consistent with `lengthSquared` i guess? and maybe in `PascalCase` too, to be consistent with the other static methods